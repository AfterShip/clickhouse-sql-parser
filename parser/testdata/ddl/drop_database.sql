DROP DATABASE IF EXISTS datbase_name;
