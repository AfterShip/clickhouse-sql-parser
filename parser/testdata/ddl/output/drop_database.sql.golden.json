[
  {
    "DropPos": 0,
    "StatementEnd": 36,
    "Name": {
      "Name": "datbase_name",
      "QuoteType": 1,
      "NamePos": 24,
      "NameEnd": 36
    },
    "IfExists": true,
    "OnCluster": null
  }
]